### ISSUE TITLE
Create a new official release

### ISSUE THREAD

**Issue by 2br-2b** [clementine-player/Clementine#6856]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Linux
- Clementine version: Clementine 1.4 rc1

### Expected behaviour / actual behaviour

Expected behavior: a new official release every so often

Actual behavior: the last official release was in 2016

### Steps to reproduce the problem (only for bugs)

Go to Github releases and see that the latest official release is from 2016

**Comment by ssrublev** [comment]
Also please provide support for Fedora 33 which is released and rpms should be fc33 but I see only fc32 and fc31

**Comment by ssrublev** [comment]
In Fedora 33 I can't install latest "clementine-1.4.0rc1-2.384.g41513527c.fc32.x86_64.rpm" it says:

Error: 
 Problem: conflicting requests
  - nothing provides libprotobuf.so.22()(64bit) needed by clementine-1.4.0rc1-2.384.g41513527c.fc32.x86_64
  - nothing provides libcdio.so.18()(64bit) needed by clementine-1.4.0rc1-2.384.g41513527c.fc32.x86_64
  - nothing provides libcdio.so.18(CDIO_18)(64bit) needed by clementine-1.4.0rc1-2.384.g41513527c.fc32.x86_64

**Comment by jbroadus** [comment]
@ssrublev Fedora 33 builds are now available.

**Comment by ssrublev** [comment]
Thank you! My next try was successful, 1.4.0rc1-2.388.g6a6ef729e.fc33 installed and working

**Comment by ssrublev** [comment]
A question about OS integration I feel unsuitable for separate issue but maybe someone can help.

I largely use media keys (play-pause/prev/next) with Clementine. In Fedora. But sometimes other applications intercept media keys. Especially VLC and Chromium with Youtube. And I can't get them back in Clementine. To get them back, I have to manually restart Clementine. I have even created a bash script that `killall clementine && clementine` But this is too rude and I guess there should be more civilized way that is also faster. Some mediakeys-related command

**Comment by ssrublev** [comment]
> Right now, I use [Strawberry](https://github.com/strawberrymusicplayer/strawberry) (which seems to be an actively-maintained form of Clementine.

Oh thank you very much for this comment

**Comment by luzpaz** [comment]
Is there a soft roadmap to the next release ?

**Comment by dontfwiththestream** [comment]
> Is there a soft roadmap to the next release ?

yes here https://github.com/strawberrymusicplayer/strawberry

**Comment by matkoniecz** [comment]
> Right now, I use Strawberry (which seems to be an actively-maintained form of Clementine.

Not exactly, it is a cut down version that misses functionality of interest for me. That is present in Clementine.

You just need to get used to official releases are now called pre-releases (no idea why such confusing strategy ended being used and why maintainers ceased to make official releases but Clementine works fine for me...)
